src/internal/packager/images/pull.go

```diff
@@ -308,6 +311,20 @@ func (i *ImageConfig) PullImage(src string, spinner *message.Spinner) (img v1.Im
 			return nil, false, fmt.Errorf("failed to load image from docker daemon: %w", err)
 		}
 	} else {
+		var idx v1.IndexManifest
+		if err := json.Unmarshal(manifest, &idx); err != nil {
```

Comment:
Is there a reason to not return the err from `json.Unmarshal` here? - It could be appended to the "lang.Err*" so we don't lose any context 🤔